From dfd0508078a54e3a32dac0abfab2c7f815f91c91 Mon Sep 17 00:00:00 2001
From: Yoshihito Ogawa <yoshihito.ogawa.kc@renesas.com>
Date: Thu, 22 Jan 2026 19:31:15 +0900
Subject: [PATCH 3/6] workaround: home-panel-demo: disable the music player

Noise issues were observed during music playback.
As a temporary workaround, the music player has
been disabled until the problem is resolved.

Signed-off-by: Yoshihito Ogawa <yoshihito.ogawa.kc@renesas.com>
---
 .../home_panel_demo/src/pages/dashboard.js        | 15 +++++----------
 1 file changed, 5 insertions(+), 10 deletions(-)

diff --git a/sample_app/chromium/home_panel_demo/src/pages/dashboard.js b/sample_app/chromium/home_panel_demo/src/pages/dashboard.js
index 79bb37a..32194f5 100644
--- a/sample_app/chromium/home_panel_demo/src/pages/dashboard.js
+++ b/sample_app/chromium/home_panel_demo/src/pages/dashboard.js
@@ -11,7 +11,6 @@ export class DashboardPage extends LitElement {
   render() {
     return html`
       <div class="container">
-        <music-widget class="music-player" title="Music Player"></music-widget>
         <icon-widget
           class="image-gallery"
           title="Image Gallery"
@@ -326,25 +325,21 @@ export class DashboardPage extends LitElement {
       height: 80%;
       transition: all 500ms ease;
     }
-    .music-player {
-      grid-column: 2 / span 3;
-      grid-row: 1 / span 2;      
-    }
     .image-gallery {
-      grid-column: 5 / span 2;
+      grid-column: 4 / span 2;
     }
     .video-player {
-      grid-column: 7 / span 2;
+      grid-column: 6 / span 2;
     }
     .home-automation {
-      grid-column: 9 / span 2;
+      grid-column: 8 / span 2;
     }
     .weather-info {
-      grid-column: 5 / span 4;
+      grid-column: 4 / span 4;
       grid-row: 2;
     } 
     .camera {
-      grid-column: 9 / span 2;
+      grid-column: 8 / span 2;
       grid-row: 2;
     }
 
-- 
2.34.1

